frontend/src/pages/ContactUs/ContactUs.module.scss

```diff
@@ -1,301 +1,303 @@
 .contact-section {
-	padding: 0 20px;
+  padding: 0 20px;
```

Comment:
also for formatting, we use tab space instead of a single space everywhere, so can you please fix it in all the css files present in this PR?